features/configuring_responses/mixed_responses.feature

```diff
@@ -8,6 +8,9 @@ Feature: Mixed responses
   use a `lambda` or similar with the same arity as your method but you can use a `proc` if you
   do not care about arity (e.g. when raising).
 
+  Note: You can also use `and_invoke` to yield. In this case, use a `proc` with a `&block`
+  parameter.
```

Comment:
I would instead rework the comment above, e.g. delete this and change the above to:

```
  Note: The invoked callable will be supplied the calls arguments, including any blocks (so `yield`
  et al will be supported). It is recommended to use a `lambda` or similar with the same arity
  as your method but you can use a `proc` if you do not care about arity(e.g. when raising).
```